tests/Tools/vmOpsTool/TestVirtualMachine.java

```diff
@@ -0,0 +1,8 @@
+class TestVirtualMachine {
+
+    String Name;
+    String NewSnapshot;
```

Comment:
Let us not do this way to update test resource. Instead you can add an xml in the tests folder and that xml will contain necessary resource names.  Test methods will read from there, so any contributor can update as per his needs.